### PR TITLE
feat: add optional background color for big gcode thumbnails

### PR DESCRIPTION
**Pull request by meteyou**
## Description

This PR adds a optional background color to gcode thumbnails.

## Related Tickets & Documents

fixes: #1507 

## Mobile & Desktop Screenshots/Recordings

Start print dialog:
image

Status panel:
image

Gcode list:
image

Settings:
image

## [optional] Are there any post-deployment tasks we need to perform?

none

### CODE REVIEW

File: src/locales/en.json [hunk 1]
```diff
@@ -1055,6 +1055,7 @@
         "UiSettingsTab": {
             "BedScrewsDialog": "Bed Screws Dialog",
             "BedScrewsDialogDescription": "Display helper dialog for BED_SCREWS_ADJUST.",
+            "BigThumbnailBackground": "Large thumbnail background",
```
Comment (dw-0) [inline src/locales/en.json:1058]:
```suggestion
            "BigThumbnailBackground": "Large thumbnail background color",
```

**Review by dw-0 — CHANGES_REQUESTED** [review]

**Comment by EmJay276** [comment]
Is it possible to add an "autodetection" of the background color based on the image color?
Dark picture => light background
Bright picture => dark background

**Comment by EmJay276** [comment]
@meteyou 
This here seems to work without a big library
https://stackoverflow.com/questions/38211798/detect-if-image-is-dark-light-and-addclass-dark-light-to-parent

**Comment by meteyou** [comment]
@EmJay276 hmm... that could work, but is a a bigger rework. can you please open a new FR with this link? so i can add this for the next release. for this release, i would just merge this PR and extend it with the next one.

**Review by dw-0 — APPROVED** [review]

**Pull request merged**